CSScomb.py

```diff
@@ -104,6 +104,7 @@ def is_css(self):
 
     def is_scss(self):
         return self.view.scope_name(0).startswith('source.scss')
+            or self.view.scope_name(0).startswith('source.sass')
```

Comment:
Oh, wow, thank you!
But you know what, there should actually be `def is_sass()` returning `sass` as a syntax name, not `scss`.